src/styles/blocks/header.scss

```diff
@@ -0,0 +1,94 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  font-size: 12px;
+  font-weight: 500;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo-link {
+  width: 40px;
+  height: 40px;
+  line-height: 60px;
+}
+
+.nav {
```

Comment:
It's not critical here, but theoretically `nav` is a separate BEM block, so it should be in a separate file according to BEM rules. Same for logo-link (or make it element of header block)